### DISCUSSION TITLE
Unable to setup via docker

### DISCUSSION THREAD

**Discussion by PuppyLover101** [meisnate12/Plex-Meta-Manager#43] [Q&A] [answered]
Hi there,

I'm having a bit of trouble setting this program up. Could someone please point me in the right direction? I am using unRAID and am attempting to setup a docker.

The docker command is: `root@localhost:# /usr/local/emhttp/plugins/dynamix.docker.manager/scripts/docker run -d --name='PlexMetaManager' --net='bridge' -e TZ="America/New_York" -e HOST_OS="Unraid" -e 'run'='True' -e 'divider'='=' -e 'width'='100' -v '/mnt/user/appdata/plexMetaManager':'/config':'rw' -v '/mnt/user/data/':'/mnt/user/data/':'rw' 'meisnate12/plex-meta-manager`

My config and movies yml files are as followed:
https://pastebin.com/raw/6RZs0UU6
https://pastebin.com/raw/gLURwnv0

Upon running the program, I only see this in the logs:

![image](https://user-images.githubusercontent.com/21270691/108266645-57cb9880-7138-11eb-8ea9-966180496c0e.png)

What have I done wrong?

Thank you,

**Comment by meisnate12** [comment, marked as answer]
so its working as intended by default the program will run at 3AM each day if you want to immediately run it add --run to the end of your docker command and then each time you start the container it will run
